src/txproto.c

```diff
@@ -224,6 +225,43 @@ AVBufferRef *tx_encoder_create(
     return NULL;
 }
 
+int tx_encoder_set_bitrate(
+    TXMainContext *ctx,
+    AVBufferRef *encoder,
+    int bitrate
+) {
+    AVDictionary *commands = NULL;
+    int err = 0;
+
+    err = av_dict_set(&commands, "command", "set_bitrate", 0);
+    if (err < 0) {
+        sp_log(ctx, SP_LOG_ERROR, "av_dict_set() failed: %s!", av_err2str(err));
+        return err;
+    }
+
+    err = av_dict_set_int(&commands, "bitrate", bitrate, 0);
+    if (err < 0) {
+        sp_log(ctx, SP_LOG_ERROR, "av_dict_set_int() failed: %s!", av_err2str(err));
+        return err;
+    }
```

Comment:
why not a single entry? this should be enough and is closer to how filter commands operate

```suggestion
    err = av_dict_set_int(&commands, "bitrate", bitrate, 0);
    if (err < 0) {
        sp_log(ctx, SP_LOG_ERROR, "av_dict_set_int() failed: %s!", av_err2str(err));
        return err;
    }
```

the entries of the dictionary can be iterated upon by the encoder to figure out what should be changed

this also opens the door to multiple options being modified in a single command